### PR TITLE
Fix. typos

### PR DESCRIPTION
**Pull request by droidbg**
## Description

This PR fixes typos in 
- `date_picker.dart`
- `date_picker_theme.dart`
- `dropdown.dart`

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter/lib/src/cupertino/date_picker.dart [hunk 1]
```diff
@@ -208,7 +208,7 @@ enum _PickerColumnType {
 ///  * US-English: `| July | 13 | 2012 |`
 ///  * Vietnamese: `| 13 | Tháng 7 | 2012 |`
 ///
-/// Can be used with [showCupertinoModalPopup] to display the picker modally at
+/// Can be used with [showCupertinoModalPopup] to display the picker modal at
```
Comment (goderbauer) [inline packages/flutter/lib/src/cupertino/date_picker.dart:211]: "modally" was not a typo here. It means "in a modal way". Can you undo this change?

**Review by goderbauer — COMMENTED** [review]
LGTM after comment is resolved

File: packages/flutter/lib/src/cupertino/date_picker.dart (hunk 1 above)
Comment (droidbg) [reply packages/flutter/lib/src/cupertino/date_picker.dart]: Thanks for the review, I have changed it back to `modally` .

**Review by goderbauer — APPROVED** [review]
LGTM

**Review by chunhtai — APPROVED** [review]
LGTM

**Review by jonahwilliams — APPROVED** [review]
LGTM
